server/db/index.js

```diff
@@ -1,21 +1,22 @@
 const db = require('./db')
-const { Op } = require('sequelize')
-const models = require('./models')
 const {
-  User,
   StretchAnswer,
-  Stretch,
   Comment,
   Cohort,
-  CohortStretch,
   Category,
-  CohortUser
-} = models
-
-require('./models/User')
```

Comment:
I think this is the reason why we were running into issues yesterday. I accidentally required ``./models/User`` instead of ``./methods/User``.

I don't think you need to make the re-declarations in your current file.